src/Symfony/Component/PropertyInfo/Extractor/ReflectionExtractor.php

```diff
@@ -107,6 +109,13 @@ public function getTypes($class, $property, array $context = array())
         if ($fromAccessor = $this->extractFromAccessor($class, $property)) {
             return $fromAccessor;
         }
+
+        if (
```

Comment:
`$context['enable_constructor_extraction'] ??  $this->enableConstructorExtraction`?

Reply:
It's not the same, because if one want to **disable** it, with your code it will fallback on the `enableConstructorExtraction` value.

Reply:
You are right. I fixed it.